VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -79,6 +82,14 @@
         COURSE
     }
 
+    public static CourseDiscussionPostsThreadFragment newInstance(Bundle args) {
```

Comment:
The usual purpose for creating the `newInstance()` factory method is to provide the parameters directly, and let it set the arguments itself. If it's only task is to call the `setArguments()` method on a provided `Bundle`, then there's not much point to having these methods.